spring-cloud-previews/google-cloud-document-ai-spring-starter/src/main/java/com/google/cloud/documentai/v1/spring/DocumentProcessorServiceSpringAutoConfiguration.java

```diff
@@ -180,6 +180,13 @@ public DocumentProcessorServiceSettings documentProcessorServiceSettings(
           .listProcessorTypesSettings()
           .setRetrySettings(listProcessorTypesRetrySettings);
 
+      RetrySettings getProcessorTypeRetrySettings =
```

Comment:
Corresponding to `document_processor_service.proto` [change](https://github.com/googleapis/googleapis/commit/bd5f9343ccec979c047c23f43509903a5563633a)